### PR TITLE
Uploading the files

### PR DESCRIPTION
**Pull request by siddghosh108**
Providing my reviews on the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  </head>
+
+  <body>
+    <!-- Header Part Start  -->
+    <header>
+      <nav id="navbar">
+        <div class="nav-item"><a href="#welcome-section"> About</a></div>
+        <div class="nav-item"><a href="#projects"> Work</a></div>
+        <div class="nav-item"><a href="#contact"> Contact</a></div>
+      </nav>
```
Comment (siddghosh108) [inline index.html:24]: I suppose you can use ul element for its list of links inside the nav element

File: index.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+
+  <body>
+    <!-- Header Part Start  -->
+    <header>
```
Comment (siddghosh108) [inline index.html:19]: I will suggest you can follow up with a main element after the header element

File: index.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            <img src="./img/tic-tac-toe.png" alt=" Tic Tac Toe Game" />
+          </a>
+          <p>Tic Tac Toe Game</p>
+        </div>
+      </section>
+
+      <section id="button">
+        <button>
+          <a href="https://codepen.io/FreeCodeCamp/"> Show all ></a>
+        </button>
```
Comment (siddghosh108) [inline index.html:88]: The button element could have been wrapped in a form element for better accessibility and to associate it with its corresponding action

File: style.css [hunk 4]
```diff
@@ -0,0 +1,164 @@
+* {
```
Comment (siddghosh108) [inline style.css:1]: The CSS code does not have any syntax errors. However, it's important to note that the height property in the #navbar and #welcome-section selectors is specified without a unit, which is not valid. It should have a unit such as px, em, or % specified. Additionally, the justify-content property in the #navbar selector is set to flex-end, which may cause the navbar links to align to the right side of the navbar. If this is not intended, justify-content: space-between or justify-content: center may be better alternatives.

**Review by siddghosh108 — COMMENTED** [review]
You have done a great job, just some more suggestions from my end may help you. Please do let me know your views

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html:24]: - [ Optional ] Kindly I recommend you to always includes an explanation, a "why", for each requested change.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html:19]: - [ Optional ] Kindly I recommend you to always includes an explanation, a "why", for each requested change.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @siddghosh108   👏 👏,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

## Highlights

- [x] Performed a code review on a past project ✔️
- [x] Used the GitHub inline comments feature ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**